frontstage/controllers/party_controller.py

```diff
@@ -438,21 +430,28 @@ def get_survey_list_details_for_party(respondent: dict, tag: str, business_party
 
     # This is a dictionary that will store all the data that is going to be cached instead of making multiple calls
     # inside the for loop for get_respondent_enrolments.
-    cache_data = {"collexes": dict(), "cases": dict()}
+    cache_data = {"cases": dict()}
     redis_cache = RedisCache()
 
     # Populate the cache with all non-instrument data
     caching_data_for_survey_list(cache_data, surveys_ids, business_ids, tag)
 
-    enrolments = get_respondent_enrolments_for_started_collex(enrolment_data, cache_data["collexes"])
+    #  Populate the enrolments by creating a dictionary using the redis_cache
+    enrolments = get_respondent_enrolments_for_started_collex(
+        enrolment_data,
+        dict((survey_id, redis_cache.get_collection_exercises_by_survey(survey_id)) for survey_id in surveys_ids),
+    )
     for enrolment in enrolments:
         business_party = redis_cache.get_business_party(enrolment["business_id"])
         survey = redis_cache.get_survey(enrolment["survey_id"])
 
         # Note: If it ever becomes possible to get only live-but-not-ended collection exercises from the
         # collection exercise service, the filter_ended_collection_exercises function will no longer
         # be needed as we can request what we want instead of having to filter what we get.
-        live_collection_exercises = filter_ended_collection_exercises(cache_data["collexes"][survey["id"]])
+
+        live_collection_exercises = filter_ended_collection_exercises(
+            redis_cache.get_collection_exercises_by_survey(survey["id"])
```

Comment:
If you do the suggestion above, you can change this to 
```python
live_collection_exercises = filter_ended_collection_exercises(collection_exercises[survey["id"]])
```
this will prevent a call to redis.  While redis is fast, getting a local variable is much faster than getting it from redis